workerapi/worker.thrift

```diff
@@ -0,0 +1,38 @@
+enum Status {
+  UNKNOWN = 0
+  READY = 1
+  PENDING = 2
+  FAILURE = 3
```

Comment:
OK, so, maybe I'm wrong and FAILURE means something else.

Let me make an analogy:
when you use the Postal Service to apply to college, you don't see if you got accepted or denied. You just get the letter. From the Worker's perspective, exitcode 0 and exitcode 1 are the same: they're different payloads for a completed process.

I think our API should reflect that. To us, processes don't fail or succeed, they complete.